### ISSUE TITLE
How to get started?

### ISSUE THREAD

**Issue by thosas** [HackTheCityio/IndependentLiving#1]
Forgive me if I am just being impatient, but is there anything I can do to help get this started?  I know it's not quite been a week yet, but would be nice to start seeing something happen!

**Comment by xyz99p** [comment]
Hi I don’t know if you are still interested in this topic but it is an area I am very interested in.
I have some hardware, Comms and limited software knowledge and would be keen to share ideas.

**Comment by HackTheCityio** [comment]
Hey,

Sure. Are you living in Swansea? Could you make the HackTheCity meetup next week?

Cheers

— Paul

**Comment by xyz99p** [comment]
Yes planning to go only found out about it last week, see you there.

Kevin

**Comment by HackTheCityio** [comment]
Cool, see you there!

**Comment by xyz99p** [comment]
Hi sorry I couldn’t make it today, how did the session go?

Kevin
